daemon/oci_linux.go

```diff
@@ -1023,13 +1003,11 @@ func WithSysctls(c *container.Container) coci.SpecOpts {
 // WithUser sets the container's user
 func WithUser(c *container.Container) coci.SpecOpts {
 	return func(ctx context.Context, _ coci.Client, _ *containers.Container, s *coci.Spec) error {
-		uid, gid, additionalGids, err := getUser(c, c.Config.User)
+		var err error
+		s.Process.User, err = getUser(c, c.Config.User)
 		if err != nil {
```

Comment:
nit: just `return err` is sufficient, no `if` is needed.

Reply:
oh! 🤦 good catch, overlooked that